src/kirby/components/chart/chart-types.ts

```diff
@@ -0,0 +1,4 @@
+export enum ChartTypes {
```

Comment:
Brug ental for Enums:
```suggestion
export enum ChartType {
```